josemak25/hairx-client
# Rf fix signup screens

#### What does this PR do?

Make changes to signup screens

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`
![IMG_2540](https://user-images.githubusercontent.com/51112934/80036866-79dd9580-84ea-11ea-99a6-8c8d797525d6.PNG)
![IMG_2539](https://user-images.githubusercontent.com/51112934/80036871-7cd88600-84ea-11ea-8950-fd6ae3c2d1f9.PNG)
![IMG_2538](https://user-images.githubusercontent.com/51112934/80036873-7d711c80-84ea-11ea-82ea-60dbb3f4f02e.PNG)




javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

#### Screenshots (if appropriate)
